frontend/insurance/src/app/shared/components/edit-claim/edit-claim.component.ts

```diff
@@ -160,7 +160,7 @@ export class EditClaimSharedComponent implements OnInit {
         name: this.insurerForm.controls['insurerName'].value
           ? this.insurerForm.controls['insurerName'].value
           : this.claim.insurer.name,
-        has_gap: this.insurerForm.controls['gap'].value
+        has_gap: Boolean(this.insurerForm.controls['gap'].value)
```

Comment:
You're mapping a string to a boolean here, and that string is either a "Yes" or "No", right? You should probably just switch this to a radio button of true or false.

Reply:
or just use the correct property of checkboxes... You don't look up value, you look up `checked`. That's the boolean

Reply:
That was the plan but the way you made all these forms makes no sense compared to the way we created the form for New-Claim-Component.